src/moonstone-samples/lib/AccessibilitySample.js

```diff
@@ -23,32 +24,56 @@ module.exports = kind({
 	labelText : 'Label',
 	hintText : 'Hint',
 	components: [
-		{kind: Header, name: 'header', content: 'Accessibility', type: 'small', components: [
+		{kind: Header, name: 'header', content: 'Accessibility', titleBelow: 'Unusual case sample for Accessibility', type: 'small', components: [
 			{name: 'labelButton', kind: Button, small: true, minWidth: false, content: 'Set Label', ontap: 'labelButtonTapped'},
 			{name: 'hintButton', kind: Button, small: true, minWidth: false, content: 'Set Hint', ontap: 'hintButtonTapped'},
-			{name: 'toggle', kind: ToggleButton, small: true, toggleOnLabel: 'disabled is on', toggleOffLabel: 'disabled is off', ontap: 'disabledTapped'},
-			{name: 'reload', kind: Button, small: true, content: 'Reload', ontap: 'reload'}
+			{name: 'toggle', kind: ToggleButton, small: true, toggleOnLabel: 'disabled is on', toggleOffLabel: 'disabled is off', ontap: 'disabledTapped'}
 		]},
 		{name: 'container', kind: Scroller, fit: true, components: [
 			{classes: 'moon-1v'},
-			{kind: Divider, content: 'Icons: '},
-			{kind: Icon, icon: 'play', small: false, ontap: 'buttonTapped', accessibilityLabel: 'play'},
-			{kind: Icon, src: '@../assets/icon-list.png', ontap: 'buttonTapped', accessibilityLabel: 'icon List'},
-			{kind: Item, classes: 'moon-hspacing', ontap: 'buttonTapped', accessibilityLabel: 'selectable icon List', components: [
-				{content: 'Selectable Item', accessibilityDisabled: true},
+			{kind: Control, tag: 'br'},
+			{kind: Divider, content: 'Case 1: Non spotlight'},
+			{kind: Icon, icon: 'play', small: false, ontap: 'buttonTapped'},
+			{kind: Icon, src: '@../assets/icon-list.png', ontap: 'buttonTapped'},
```

Comment:
Yes. My intent is that default Icon can not read any string when clicked, but can read label or hint string if accessibilityLabel or accessibilityHint is added.